_lessons/python-scatter/00-introduction.md

```diff
@@ -81,17 +81,17 @@ python scatter.py
 ```
 or by submitting a job to the scheduler. On Mahuika
 ```
-srun python scatter.py
+sbatch scatter.sl
 ```
-where you might have to pass `--account="myAccount"` as well as other options to the `srun` command. [See](https://support.nesi.org.nz/hc/en-gb/articles/360000359576-Slurm-Usage-A-Primer) for more information.
+where you might have to edit `scatter.sl` and add `#SBATCH --account="myAccount"` as well as other options. [See](https://support.nesi.org.nz/hc/en-gb/articles/360000359576-Slurm-Usage-A-Primer) for more information.
 
 ### Adjusting the domain size and contour resolution
 
 As you improve the performance of the code, you'll find it useful to increase the problem resolution. This can be done by passing command line options to `scatter.py`. Type `python scatter.py -h` to see the full list of options. The options that control the grid size are `-nx # -ny #` for the number of cells in the x and y direction. Option `-nc #` sets the number of segments. 
 
 The default values are `-nx 128`, `-ny 128` and `-nc 128`. The execution time scales linearly with the values of options `-nx`, `-ny` and `-nc`. For example:
 ```
-srun python scatter.py -nx 256 -ny 256 -nc 512
+python scatter.py -nx 256 -ny 256 -nc 512
```

Comment:
@pletzer do we actually want people to submit to the queue or is it OK for them all to run interactively? If we want them to submit to the queue we should probably say here to edit the command in scatter.sl and run sbatch to submit to queue, otherwise they will just all run the command on the login node.